core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -7158,6 +7158,15 @@ public void testJsonPretty() {
         .returns("A=29; B=35; C=37; D=36\n");
   }
 
+  @Test
```

Comment:
remove this test. JdbcTest is too large already. this is not sufficiently important.

add instead to one of the .iq tests.

Reply:
Removed test from JdbcTest, Tried adding test to 
# misc.iq
# XML
values ExtractValue('<a>c</a>', '//a');
EXPR$0
c
!ok

--
Test case failed with : java.sql.SQLException: Error while executing SQL "values ExtractValue('<a>c</a>', '//a')": From line 1, column 8 to line 1, column 38: No match found for function signature EXTRACTVALUE(<CHARACTER>, <CHARACTER>)


It would be great if you could help me and point to some example :)

Reply:
Done :)